src/components/messages/TextMessage.vue

```diff
@@ -99,15 +99,16 @@ export default {
   line-height: 1.4;
   -webkit-font-smoothing: subpixel-antialiased;
   box-shadow: 0 2px 4px 0 rgba(85, 87, 85, 0.5);
-  overflow-wrap: break-all;
+  overflow-wrap: break-word;
   hyphens: auto;
-  word-break: break-all;
+  word-break: break-word;
```

Comment:
`word-break: break-word` has been deprecated and so should not be used, it also implicitly overwrites the behaviour of `overflow-wrap` so even if we did use `word-break: break-word` we would then not need `overflow-wrap` at all:
<img width="513" alt="image" src="https://github.com/artificialsolutions/teneo-web-chat/assets/90412464/8ec2f919-6eaf-43f5-b275-9a54048aec8b">
https://developer.mozilla.org/en-US/docs/Web/CSS/word-break

Reply:
will remove